File: src/main/java/ti/proyectoinia/dtos/HumedadReciboDto.java
package ti.proyectoinia.dtos;

import lombok.Data;
import ti.proyectoinia.business.entities.HumedadLugar;

@Data
public class HumedadReciboDto {

    private Long id;

    private HumedadLugar lugar;

    private Integer numero;
}

File: frontend/src/models/HumedadRecibo.dto.ts
import { HumedadLugarDto } from './HumedadLugar.dto';

export interface HumedadReciboDto {
  id: number | null;
  reciboId: number | null;
  numero: number | null;
  lugar: HumedadLugarDto | null;
}